_test/test_experiment/test_experiment_loadsave.m

```diff
@@ -39,6 +39,14 @@ function test_loadsave_single_run_and_sqw(obj)
             ld1 = load(wkfile);
             % tmp_sqw is checked that it now does not have the marker field
             assertTrue( isa(ld1.tmp_sqw.experiment_info, 'Experiment') );
+            assertEqual( ld1.tmp_sqw.experiment_info.detector_arrays, ...
+                             tmp_sqw.experiment_info.detector_arrays );
+            assertEqual( ld1.tmp_sqw.experiment_info.instruments, ...
+                             tmp_sqw.experiment_info.instruments );
+            assertEqual( ld1.tmp_sqw.experiment_info.samples, ...
+                             tmp_sqw.experiment_info.samples );
+            assertEqual( ld1.tmp_sqw.experiment_info.expdata, ...
+                             tmp_sqw.experiment_info.expdata );
```

Comment:
all these checks are unnecessary. Last comparison do compare them all unless you have redefined the comparison operator